notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Harvester.scala

```diff
@@ -34,9 +34,14 @@ trait HarvesterRequestHandler extends Logging {
     throwables.broadcastTo(logErrors, cloudwatch.sendFailures(env.stage, platform))
   }
 
-  def firebaseSink(notification: Notification, range: ShardRange): Sink[IO, String] = {
+  def firebaseSinkBuilder(notification: Notification, range: ShardRange): Sink[IO, (String, Platform)] = {
```

Comment:
why rename this  method and not the aprns one on L54

Reply:
There was a name conflict with L75

Reply:
Fair enogh